javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -125,6 +125,7 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
       agentBuilder =
           agentBuilder
               .with(AgentBuilder.RedefinitionStrategy.RETRANSFORMATION)
+              .with(AgentBuilder.RedefinitionStrategy.DiscoveryStrategy.Reiterating.INSTANCE)
```

Comment:
cc: @tylerbenson, I think this will fix that Cassandra test for you (https://github.com/DataDog/dd-trace-java/pull/1802)

Reply:
Can you add to the PR description what happened?

Reply:
👍 done, it's a weird one, i can add more details if it is unclear

Reply:
Hmm - not too clear to me still but not sure if it ever would. Main question for me is why we didn't need that before all of our package renaming. Is it because instead of `io.opentelemetry.javaagent` we switched to the more generic `io.opentelemetry`?

Reply:
oh ya, scroll up just a bit, this code is inside of a conditional `if (log.isDebugEnabled()) {` which wasn't triggered until I fixed the debug logging in `AgentTestRunner`

Reply:
as far as why we didn't need `RedefinitionStrategy.DiscoveryStrategy.Reiterating`, it's probably still not needed in prod situations because there aren't many classes already loaded when we install the agent, but it's different in tests where we apply instrumentation to some already loaded classes. in particular we already load guava classes, which is what caused the test issue with the Cassandra ListenableFuture test